content/platforms/kubernetes/stability.md

```diff
@@ -1,53 +1,46 @@
 ---
 Title: Managing pod stability
-description: Pods can be evicted from a node for various reasons - including
- being preempted by other scheduling requests. There are various ways for
- controlling this behaviour.
+description: You can use quality of service, priority class, eviction thresholds and resource monitoring
+    to maintain cluster node pod stability.
 weight: 90
 alwaysopen: false
 categories: ["Platforms"]
 aliases:
 ---
+Kubernetes clusters manage the allocation of system resources and can evict pods to release system resources.
+Here are some ways that you can configure the Redis Enterprise node pods to maintain pod stability:
 
-Redis Enterprise cluster node pods can be evicted from a node for variety
-of ways that can be controlled by configuring the cluster or the node pods
-correctly for the desired outcome.
+## Guaranteed Quality of Service
 
-## Guaranteeing quality of service
+At pod creations, Kubernetes assigns the pod one of three [quality of service classes](https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/):
+Guaranteed, Burstable, and Best Effort.
+You can assign the Guaranteed class to the Redis Enterprise node pods.
 
-When Kubernetes creates a pod it assigns it one of three [quality of service
-classes](https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/):
-Guaranteed, Burstable, and Best Effort. For Redis Enterprise node
-pods we want all the pods to be assigned the Guaranteed class.
+To assign a quality of service class:
 
-To do so, each operator managed pods must have:
+ * Every container in the pod must have a memory limit and a memory request, and they must be the same.
+ * Every container in the pod must have a CPU limit and a CPU request, and they must be the same.
 
- * Every Container must have a memory limit and a memory request, and they must be the same.
- * Every Container must have a CPU limit and a CPU request, and they must be the same.
+These requirements are met in the default version of your Redis Enterprise cluster CRD,
```

Comment:
This is not what happens. The CRD does not contain requests and limits unless you put them in there.

Reply:
I fixed this.